pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -98,6 +98,10 @@ func init() {
 					Verbs:     sets.NewString("update"),
 					Resources: sets.NewString("builds"),
 				},
+				{
+					Verbs:     sets.NewString("create"),
+					Resources: sets.NewString("builds/docker", "builds/source", "builds/custom"),
+				},
```

Comment:
so this is needed because the build controller *updates* builds? If so, it would make more sense to have this be an update permission